data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/SessionEvent.avsc

```diff
@@ -50,8 +50,8 @@
     {
       "name": "payload",
       "type": [
-        "net.corda.data.flow.event.session.SessionInit",
```

Comment:
Do we need to remove https://github.com/corda/corda-api/blob/CORE-16860/counterparty-info/data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/session/SessionInit.avsc as well?